docs/api/puppeteer.elementhandle.uploadfile.md

```diff
@@ -31,3 +31,5 @@ Promise&lt;void&gt;
 ## Remarks
 
 This will not validate whether the file paths exists. Also, if a path is relative, then it is resolved against the [current working directory](https://nodejs.org/api/process.html#process_process_cwd). For locals script connecting to remote chrome environments, paths must be absolute.
+
+Not supported with [WebDriver BiDi](https://pptr.dev/faq#q-what-is-the-status-of-cross-browser-support).
```

Comment:
I know it's auto-generated, but it can be useful to have a special section "WebDrive BiDi support level" to docs. Is it feasible?

Reply:
what would be in that section?

Reply:
"Supported in protocols (CDP, WebDriver BiDi)" / "WebDriver BiDi status" ?

Reply:
do you mean like listing every method that is supported?

Reply:
In `.md` file of every method. Or only in methods which don't fully support BiDi (this PR does this, AFAIA).

Reply:
ah, no, we cannot move it to a separate section since tsdoc does not support custom tags/templates. I thought you meant a top-level page about the webdriver bidi status.